go/noise/noise.go

```diff
@@ -57,6 +59,22 @@ func ToKind(n Noise) Kind {
 	return GaussianNoise
 }
 
+// ConfidenceIntervalInt64 holds upper and lower bounds in int64 for the confidence interval.
+type ConfidenceIntervalInt64 struct {
+	LowerBound, UpperBound int64
+}
+
+// ConfidenceIntervalFloat64 holds upper and lower bounds in float64 for the confidence interval.
+type ConfidenceIntervalFloat64 struct {
+	LowerBound, UpperBound float64
+}
+
+// toConfidenceIntervalInt64 returns ConfidenceIntervalInt64 struct after rounding the upper
```

Comment:
How about "toConfidenceIntervalInt64 converts a ConfidenceIntervalInt64 struct to ConfidenceIntervalFloat64 by rounding the lower and upper bounds."?

Reply:
@miracvbasaran Yep, sgtm! I think it should be from float64 to int64, right?